pkg/cloudinit/network.go

```diff
@@ -89,9 +132,23 @@ func (r *NetworkConfig) validate() error {
 		return ErrMissingNetworkConfigData
 	}
 	for _, d := range r.data.NetworkConfigData {
+		// refactor this when network configuration is unified
```

Comment:
please prepend `TODO` to help sonarcloud keep track of it

Reply:
Done. I'll also make an issue